.circleci/config.yml

```diff
@@ -1,30 +1,24 @@
 version: 2
 jobs:
-  build:
-
-    branches:
-      only:
-        - anchor
```

Comment:
Did you specifically mean to remove the branch tagging I had added?

Currently we only have tests in the anchor branch - so I was attempting to prevent circle from running on all the other branches.

Reply:
Shouldn't be needed since other branches won't have a circle config

Reply:
It was - which is why I tried adding it into the config. Like https://github.com/greymass/eos-voter/pull/679, which tried to build and then threw an error saying it couldn't find the circleci config:

https://circleci.com/gh/greymass/eos-voter/79?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link